README.md

```diff
@@ -162,7 +162,7 @@ For more information about the model archiver, see [Torch Model archiver for Tor
 After you archive and store the model, use the `torchserve` command to serve the model.
 
 ```bash
-torchserve --start --model-store ./model_store --models ./model_store/densenet161.mar
+torchserve --start --ncs --model-store model_store --models densenet161.mar
```

Comment:
I saw `ncs` come in here sort of randomly and w/o explanation in recent runs. What is it for? As we've somewhat described each element already, it's weird to have this setting go unmentioned.